pkg/controllers/provisioning/suite_test.go

```diff
@@ -2583,6 +2584,447 @@ var _ = Describe("Provisioning", func() {
 			})
 		})
 	})
+
+	Context("AutoRelaxMinValues", func() {
```

Comment:
Do we have consolidation testing in place for this as well? I'd want to validate scenarios like:
- When I have a NodeClaim that is fulfilling the NodePool's minValues without relaxation, I shouldn't consolidate to a cheaper node if consolidating to a cheaper node implied that I would relax the minValues